lib/jsonapi/utils.rb

```diff
@@ -245,7 +248,7 @@ def count_records(records, options)
 
     def setup_request
       @request ||=
-        JSONAPI::Request.new(
+        JSONAPI::RequestParser.new(
```

Comment:
This will break next time you bump the `jsonapi-resources` dep per https://github.com/cerebris/jsonapi-resources/commit/f8dcf8558a1c2d8018a22f74fd0514d0779b5bb9